client/extensions/targetedPublishing/Tenant.js

```diff
@@ -26,14 +28,15 @@ export default class Tenant extends Component {
 
     componentDidMount() {
         this.getRoutes();
+        this.getContentLists();
     }
 
     componentWillReceiveProps(nextProps) {
         if (nextProps.rule) {
             const rule = nextProps.rule;
             this.setState({
-                rule: rule,
-                newRule: rule
+                rule: JSON.parse(JSON.stringify(rule)),
```

Comment:
why you hack it like that?

Reply:
that is one of two ways to deeply copy an object